### ISSUE TITLE
express-validator Break validator chain

### ISSUE THREAD

**Issue by arunkillu** [ctavan/express-validator#273]
```
var input = {
            'username': {
                notEmpty: true,
                isEmail: {
                    errorMessage: 'Invalid Email.'
                }
            },
            'password': {
                notEmpty: true,
                errorMessage: 'Invalid Password' // Error message for the parameter
            }
        };

    req.checkBody(input);
    var errors = req.validationErrors();
    if (errors) {
        console.log(errors);
    }
```

If I pass username as empty output is

[ { param: 'username', msg: 'Invalid param', value: '' },
  { param: 'username', msg: 'Invalid Email.', value: '' },
  { param: 'password', msg: 'Invalid Password', value: '' } ]

how to brake the chain if anyone of the validation fails.

**Comment by gustavohenke** [comment]
When exactly do you want to stop the validation?
Is it after the first error across all validations, or do you want one validation per field, just like #235?

**Comment by arunkillu** [comment]
yes exactly as #235 , But it would be an added advantage if we can stop across all validations.

**Comment by gustavohenke** [comment]
If it's exactly like #235, then it's fixed as of v3.0.0!  
Please check the new docs regarding the result API `result.useFirstErrorOnly()`.
